### PR TITLE
Update docs to improve installation experience

### PR DESCRIPTION
**Pull request by ttuffin**
- Point users to readthedocs getting started page
- Minor grammar/formatting changes
- Remove references to eda-server

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -60,15 +60,8 @@ Features
 ===============
 Documentation
 ===============
-To learn more about using ``ansible-rulebook`` view the `Docs site <https://ansible-rulebook.readthedocs.io/>`_.
+Please refer to the `Getting Started guide <https://ansible-rulebook.readthedocs.io/en/stable/getting_started.html>`_ for instructions on how to install and use ``ansible-rulebook``.
```
Comment (Alex-Izquierdo) [inline README.rst:63]: IMO, there is no need to merge documentation and installation sections.

**Review by Alex-Izquierdo — APPROVED** [review]
IMO, there is no need to merge documentation and installation sections. Otherwise, lgtm

**Review by benthomasson — APPROVED** [review]

**Comment by ttuffin** [comment]
@Alex-Izquierdo when the [latest](https://ansible-rulebook.readthedocs.io/en/latest/getting_started.html) docs are merged the first part of the getting started guide is a link to the installation docs, so my thought was that "Getting Started" would be a good, single starting point. What do you think?

**Comment by ttuffin** [comment]
@Alex-Izquierdo @benthomasson pls re-review, I added the installation section back in.

**Review by Alex-Izquierdo — APPROVED** [review]

**Review by bzwei — APPROVED** [review]

**Pull request merged**